<launch>
  <node pkg="board_status" type="main.py" name="board_status" output="screen"/>

  <!-- Initial tokens locations -->
  <rosparam param="initial_board">
    {'1': '0', "2": '0', "3": '0', "4": '0', "5": '0',
                   "6": '0', "7": '0', "8": '0', "9": '0', "10": '0',
                   "11": '1', "12": '2', "13": '3', "14": '4', "15": '5',
                   "16": '6', "17": '7', "18": '8', "19": '9', "20": '10'}
  </rosparam>

  

  <!-- Tokens to play -->
  <rosparam param="tokens_number">
    [ '1', '2', '3', '4', '5', "6", "7", "8", "9", "10"]
  </rosparam>

  <!-- RFID IDs for the board -->
  <rosparam param="rfid_id">
    [  "3315042303", "4117136895", "2777051135", "2509992703", "88436991", "3852242175", "1972340735", "356806911", "2504166399", "2768466943"]
  </rosparam>



</launch>
